### DISCUSSION TITLE
Help with debugging on macOS - Vorta stucks on startup and does not open

### DISCUSSION THREAD

**Discussion by Vantomas** [borgbase/vorta#1507] [General]
Hello,
I got new Mac Mini and I copied the home directory from the old one to the new one. After that I have problem with Vorta which does not open anything and stuck only with one process vorta-darwin. If I try to open it in new user profile it works alright so it's something user profile related. 

I tried to use something like strace, but for dtrace I have to disable SIP and Instruments (File Activity/System trace/Time profiles) doesn't show me anything interesting. 

Can you point me in any direction how can I debug this and get why it's stucking somewhere on open, please?

**Comment by real-yfprojects** [comment]
You could try starting vorta from the terminal. The error message can hopefully be found in the output.

**Comment by Vantomas** [comment]
Unfortunately it doesn't show anything and it's stuck on something. Even CTRL+C doesn't work and I have to kill the process. I know from another Mac or a clean user profile that Vorta forks into multiple processes and that doesn't happen either.

![Snímek obrazovky 2022-12-11 v 13 33 36](https://user-images.githubusercontent.com/5727555/206903801-ef0cf94a-da5a-4ea9-9cad-6698439317ff.png)
![Snímek obrazovky 2022-12-11 v 13 32 56](https://user-images.githubusercontent.com/5727555/206903766-225dc599-ad0f-48d8-8faf-0cc247fce7b5.png)

**Reply by real-yfprojects** [reply]
Lets move this to *issues*. I created #1510 for that.
